### PR TITLE
finalize artists table for the DB with

### PR DESCRIPTION
**Pull request by metavaults**
SCRIPT THAT ADDS WORKS AND RELEASES FOREIGN KEYS TO ARTISTS TABLE

### CODE REVIEW

File: DataForSequelizeAPI/app2.js [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+            for (z = 0; z < ReadArtists.length; z++) {
+                if (obj[i].artistId == ReadArtists[z].artistId) {
+                    
+                    var word = obj[i].id.toString();
+                    ReadArtists[z].release.push(word);
+                }
+
+            }
+        
+        // SCRIPT THAT ADDS ARTISTS ID'S TO WORKS AND RELEASES
```
Comment (ShilpaGhanashyamGore) [inline DataForSequelizeAPI/app2.js]: You could remove the commented lines of code

Comment (pfent) [reply DataForSequelizeAPI/app2.js]: Yup. don't commit commented out code. That's what version control is for.

**Review by TimHenkelmann — APPROVED** [review]
👍

**Comment by sacdallago** [comment]
@LukasNavickas WHY ARE YOU SHOUTING? 😆 😆 😆 😆 😆 😆 ahahah

**Comment by kordianbruck** [comment]
@sacdallago pssst, we want to sleep :zzz: :sleeping:

**Pull request merged**